src/main/java/com/logpresso/scanner/Log4j2Scanner.java

```diff
@@ -453,7 +459,9 @@ private void fix() {
 
 		// delete backup files only if zip file is generated
 		for (File backupFile : backupFiles)
-			backupFile.delete();
+			if(!backupFile.delete()) {
+			  System.out.println("backup file cannot be deleted");
```

Comment:
User should be notified which file was problem
`System.out.println("Error:  Backup file cannot be deleted - " + backupFile.getAbsolutePath());`